snuba/web/__init__.py

```diff
@@ -27,6 +27,12 @@ def __init__(self, extra: QueryExtraData):
         self.extra = extra
 
 
+class QueryTooLongException(SerializableException):
```

Comment:
Is this naming a bit ambiguous? I first read it as "query run-time too long" and when I read "long queries" (as in the PR title) I sort of mentally translate it to "slow queries".

Don't let this block if nobody agrees

Reply:
I see, I can change it to something like `QueryStringTooLongException`?